el-gr/pyqt-gr.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+category: tool
+tool: PyQT
+filename: learnpyqt.py
```

Comment:
It should be `learnpyqt-gr.py`, otherwise it may cause conflicts when viewing the English version.